workflows/eager.nf

```diff
@@ -125,16 +132,55 @@ workflow EAGER {
         PREPROCESSING ( INPUT_CHECK.out.fastqs, adapterlist )
         ch_reads_for_mapping = PREPROCESSING.out.reads
         ch_versions          = ch_versions.mix(PREPROCESSING.out.versions)
```

Comment:
```suggestion
        ch_versions          = ch_versions.mix( PREPROCESSING.out.versions )
```
There are now several inconsistencies with white-spaces :D Maybe do that at the very end?

Reply:
I like it with, so I approve of the change - and as executive decision we format accordingly in the future ;)